src/patchers/@nextcloud/router.js

```diff
@@ -25,6 +25,10 @@ import { getRootUrl, generateFilePath as _generateFilePath } from '@desktop-modu
 
 export { linkTo, getRootUrl, generateUrl } from '@desktop-modules--@nextcloud/router'
 
+// getBaseUrl added in v3.0.0
+// On Talk Desktop same as getRootUrl, because getRootUrl is absolute already
+export const getBaseUrl = getRootUrl
```

Comment:
Now we should be careful about it, I think

Reply:
Or find a better solution to handle requests and URL...

Reply:
But I have no ideas